controllers/update_database_configuration.go

```diff
@@ -37,6 +36,10 @@ type updateDatabaseConfiguration struct{}
 
 // reconcile runs the reconciler's work.
 func (u updateDatabaseConfiguration) reconcile(ctx context.Context, r *FoundationDBClusterReconciler, cluster *fdbtypes.FoundationDBCluster) *requeue {
+	if !pointer.BoolDeref(cluster.Spec.AutomationOptions.ConfigureDatabase, true) {
```

Comment:
Do you want to log an event here, the way it used to do (on line 88 below)?

Reply:
Also, why is it not applying the cluster status and doing a require the way it used to do before this change?

Reply:
We can't log an event here since we skip all work in this reconciler. If we want to keep logging an event we would need to still perform the same operations, otherwise the reconciler can't know if the current configuration matches the desired configuration (or not). By require you mean a `requeue`? With the new approach there is no need for this, since we skip all work in this reconciler.

The new approach is an optimisation to reduce the load in the operator and on the FDB cluster.